### PR TITLE
[WIP] Stabilize integration tests

### PR DESCRIPTION
**Pull request by bastilian**

### CODE REVIEW

**Comment by ohadlevy** [comment]
thanks! I'm more than happy to close #4987 and implement it here instead :-)

File: test/integration_test_helper.rb [hunk 1]
```diff
@@ -28,8 +29,19 @@
...
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w(headless disable-gpu) }
+  )
+
+  Capybara::Selenium::Driver.new app,
+    browser: :chrome,
+    desired_capabilities: capabilities
+end
+
+Capybara.default_max_wait_time = 5
```
Comment (ohadlevy) [inline test/integration_test_helper.rb]: won't this change the current default for phantomjs too? I would suggest to keep phantomjs on our jenkins slaves, but enable developers to use the faster chrome driver.

File: test/integration_test_helper.rb [hunk 2]
```diff
@@ -28,8 +29,19 @@
   Capybara::Poltergeist::Driver.new(app, opts)
 end
 
-Capybara.default_max_wait_time = 30
-Capybara.javascript_driver = :poltergeist
+Selenium::WebDriver::Chrome.driver_path = File.join(Rails.root, 'node_modules', '.bin', 'chromedriver')
```
Comment (ohadlevy) [inline test/integration_test_helper.rb]: worth noting that in the past, I had an issue where npm chromedriver was newer then the chrome (or chromium) that was installed on slaves, but was too old for the versions you found in fedora today.

File: test/integration_test_helper.rb (hunk 1 above)
Comment (bastilian) [reply test/integration_test_helper.rb]: This is intentional. The lower wait time is to try and see if we can have this lowered.

Locally I had cases where tests seemed to simply waiting that time, idle, taking about 30 seconds and then pass. The lower wait time made those faster and still pass.

**Comment by ohadlevy** [comment]
please use redmine https://projects.theforeman.org/issues/21592 - thanks.

File: test/integration_test_helper.rb (hunk 2 above)
Comment (bastilian) [reply test/integration_test_helper.rb]: There is now `TESTDRIVER_PATH` env variable to allow setting this per environment if necessary.

**Comment by ares** [comment]
Is there something we could take from this PR without taking unfinished part? How much is missing?

**Comment by mmoll** [comment]
@bastilian could you please fix the rubocop errors and rebase? This looks almost ready...

**Comment by mmoll** [comment]
@bastilian what's the staus here? This needs at least a rebase (part of it went in with GH-6160) and fixing of the last two(?) remaining errors...

**Comment by mmoll** [comment]
@bastilian ping?

**Comment by mmoll** [comment]
so, I guess this is lost now? :disappointed:

**Comment by mmoll** [comment]
closing in favour of GH-6748

**Pull request closed without merging**